### PR TITLE
[IMP] theme_enark: revamp the theme

### PR DESCRIPTION
**Pull request by Brieuc-brd**
task-4178071
Part of task-4177978

| Before | After |
|--------|--------|
| image | enark_screenshot |

### CODE REVIEW

File: theme_enark/static/src/js/tour.js [hunk 1]
```diff
@@ -4,39 +4,44 @@ import * as wTourUtils from "@website/js/tours/tour_utils";
 
 const snippets = [
     {
-        id: 's_banner',
-        name: 'Banner',
-        groupName: "Intro",
+        id: 's_freegrid',
+        name: 'Free grid',
+        groupName: "Columns",
```
Comment (bvr-odoo) [inline theme_enark/static/src/js/tour.js:9]: not related to this line but I see a layout issue in one of the landing pages (top padding missing :wink: )
image

File: theme_enark/views/snippets/s_cta_box.xml [hunk 2]
```diff
@@ -2,9 +2,14 @@
 <odoo>
 
 <template id="s_cta_box" inherit_id="website.s_cta_box">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc o_cc2 pt-0" remove="pt80" separator=" "/>
```
Comment (bvr-odoo) [inline theme_enark/views/snippets/s_cta_box.xml:7]: `pt-0` => `pt0`

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
Good work, thanks :+1: 
Just 2 small things to change

**Comment by bso-odoo** [comment]
@robodoo delegate=bvr-odoo

**Comment by bvr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**